tests/e2e/pact/specs/live/eps.convert.spec.ts

```diff
@@ -76,22 +83,25 @@ jestpact.pactWith(
 
         const requestStr = LosslessJson.stringify(request)
         const requestJson = JSON.parse(requestStr)
+        const requestId = uuid.v4().toString().toLowerCase()
```

Comment:
We do this a lot, can we have a `uuidstring` function somewhere so we dont need to keep typing this.

Reply:
`uuid.v4()` already gives you a string and a lowercase one at that

Reply:
The function you're thinking of is in the coordinator which gets packaged separately from the PACTs, I can duplicate the function if we think it's needed. I'll remove all the `.toString().toLowerCase()` if it already does that, I just pulled it from the stuff in coordinator